File: GameServer/wwwroot/data/player.js
export class Player {
    id;
    name;
    color;
    gold;
    wood;

    /**
     * @param {string} id 
     * @param {string} name
     * @param {string} color 
     * @param {number} gold
     * @param {number} wood
     */

    constructor(id, name, color, gold, wood) {
        this.id = id;
        this.name = name;
        this.color = color;
        this.gold = gold;
        this.wood = wood;
    }
}

File: GameServer/Server/Game/Player.cs
namespace GameServer.Server.Game;

public sealed class Player
{
    public long Id { get; init; }

    public string Name { get; init; } = "";

    public string Color { get; init; } = "";

    // public int Gold { get; set; } = 0;
    // public int Wood { get; set; } = 0;
    // public DateTimeOffset CreatedAt { get; init; } = DateTimeOffset.UtcNow;
    // public DateTimeOffset LastSeenAt { get; set; } = DateTimeOffset.UtcNow;
}
